pkg/controller/certificates/signer/signer.go

```diff
@@ -73,7 +74,7 @@ func NewLegacyUnknownCSRSigningController(
 	caFile, caKeyFile string,
 	certTTL time.Duration,
 ) (*CSRSigningController, error) {
-	return NewCSRSigningController("csrsigning-legacy-unknown", capi.LegacyUnknownSignerName, client, csrInformer, caFile, caKeyFile, certTTL)
+	return NewCSRSigningController("csrsigning-legacy-unknown", "kubernetes.io/legacy-unknown", client, csrInformer, caFile, caKeyFile, certTTL)
```

Comment:
not a big issue, but why no longer reference the const here given it does still exist in v1beta1? Reducing the number of things we import/avoiding importing both v1beta1 and v1?

Reply:
yeah... I was on the fence... switched back to referencing the v1beta1 constant